File: POO JAVA/Aula11/src/aula11/Tecnico.java
package aula11;

public class Tecnico extends Aluno{

    //Atributos
    private int registro;
    
    //Metodos Acessores e Modificadores
    public int getRegistro() {
        return registro;
    }
    public void setRegistro(int registro) {
        this.registro = registro;
    }
    
    //Metodo da classe
    public void praticar(){
        System.out.println("Executando atividades praticas.");
    }
    
}

File: POO PHP/Aula11/Tecnico.php
<?php

class Tecnico extends Aluno{

    //Atributos da Classe
    private $registroProficiopnal;
    
    //Metodos Acessores e Modificadores
    public function getRegistroProficiopnal() {
        return $this->registroProficiopnal;
    }
    public function setRegistroProficiopnal($registroProficiopnal): void {
        $this->registroProficiopnal = $registroProficiopnal;
    }

    //Metodos da Classe
    public function praticar() {
        echo "<p>Praticando atividade do Curso.</p>";
    }
}
